audiorecorder/src/main/java/org/odk/collect/audiorecorder/mediarecorder/MediaRecorderRecordingResource.kt

```diff
@@ -22,13 +23,15 @@ internal abstract class MediaRecorderRecordingResource(private val mediaRecorder
     }
 
     override fun pause() {
-        if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.N) {
+        @SuppressLint("NewApi")
```

Comment:
The IDE warns us about this otherwise (even though we are performing the check).